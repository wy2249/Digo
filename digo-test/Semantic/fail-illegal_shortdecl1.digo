func digo_main() void {
  i := 2,3
}
