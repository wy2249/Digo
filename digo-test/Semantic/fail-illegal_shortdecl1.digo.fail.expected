Fatal error: exception Failure("short decl assignment mismatch: 1 variables but 2 values")
